src/App.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+export default class App extends React.Component {
+    state ={
+        result :{},
+        loading: true
+    }
+    
+   async componentDidMount(){
+       const url="http://www.omdbapi.com/?t=the&apikey=d3a8db41"
```

Comment:
Since the URL is static can we use a project level const?

Reply:
Done!